pywin32_postinstall.py

```diff
@@ -1,771 +1,4 @@
-# binary installer or uninstaller, the command line args are either
-# '-install' or '-remove'.
-
-# Important: From inside the binary installer this script MUST NOT
-# call sys.exit() or raise SystemExit, otherwise not only this script
-# but also the installer will terminate! (Is there a way to prevent
-# this from the bdist_wininst C code?)
-
-
-def verify_destination(location):
-    if not os.path.isdir(location):
-        raise argparse.ArgumentTypeError('Path "{}" does not exist!'.format(location))
-    return location
-
-
-def main():
-    import argparse
-
-    parser = argparse.ArgumentParser(
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-        description="""A post-install script for the pywin32 extensions.
-
-    * Typical usage:
-
-    > python pywin32_postinstall.py -install
-
-    If you installed pywin32 via a .exe installer, this should be run
-    automatically after installation, but if it fails you can run it again.
-
-    If you installed pywin32 via PIP, you almost certainly need to run this to
-    setup the environment correctly.
-
-    Execute with script with a '-install' parameter, to ensure the environment
-    is setup correctly.
-    """,
-    )
-    parser.add_argument(
-        "-install",
-        default=False,
-        action="store_true",
-        help="Configure the Python environment correctly for pywin32.",
-    )
-    parser.add_argument(
-        "-remove",
-        default=False,
-        action="store_true",
-        help="Try and remove everything that was installed or copied.",
-    )
-    parser.add_argument(
-        "-wait",
-        type=int,
-        help="Wait for the specified process to terminate before starting.",
-    )
-    parser.add_argument(
-        "-silent",
-        default=False,
-        action="store_true",
-        help='Don\'t display the "Abort/Retry/Ignore" dialog for files in use.',
-    )
-    parser.add_argument(
-        "-quiet",
-        default=False,
-        action="store_true",
-        help="Don't display progress messages.",
-    )
-    parser.add_argument(
-        "-destination",
-        default=sysconfig.get_paths()["platlib"],
-        type=verify_destination,
-        help="Location of the PyWin32 installation",
-    )
-
-    args = parser.parse_args()
-
-    if not args.quiet:
-        print("Parsed arguments are: {}".format(args))
-
-    if not args.install ^ args.remove:
-        parser.error("You need to either choose to -install or -remove!")
-
-    if args.wait is not None:
-        try:
-            os.waitpid(args.wait, 0)
-        except os.error:
-            # child already dead
-            pass
-
-    silent = args.silent
-    verbose = not args.quiet
-
-    if args.install:
-        install(args.destination)
-
-    if args.remove:
-        if not is_bdist_wininst:
-            uninstall(args.destination)
-
+from win32.lib.postinstall.pywin32_postinstall import main
```

Comment:
Do you mind adding a comment here explaining why this file exists at all, and how the preferred way of running it is now via `-m ...` etc? And ditto for the other.